src/controller/java/src/chip/devicecontroller/ControllerParams.java

```diff
@@ -151,9 +151,6 @@ public static class Builder {
     private Builder() {}
 
     public Builder setFabricId(long fabricId) {
-      if (fabricId < 1) {
```

Comment:
Even long can not express the full range of uint_64, but this check is still valid, right? can fabricId be a negative number?

Reply:
@yufengwangca 
As mentioned above, negative values of long type can be matched from 0x8000000000000000 to 0xFFFFFFFFFFFFFFFF of uint64_t.
In the case of nodeID and fabricID, they are used after being created as ULong type in kotlin and then changed to java long type. (Use toLong()).

e.g., https://github.com/project-chip/connectedhomeip/blob/a40963d7d571c18f4da18205b918e0c1b4e788f5/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/AddressUpdateFragment.kt#L18